webapp/packages/plugin-navigation-tree/src/NavigationTree/ElementsTree/ElementsTree.tsx

```diff
@@ -100,9 +100,18 @@ const oldStyles = css`
 `;
 
 export interface ElementsTreeProps extends IElementsTreeOptions, React.PropsWithChildren {
+  /**
+   * Specifies the root node for the elements tree, if applicable.
+   */
   root?: string;
```

Comment:
Specifies the root path for the tree. ROOT_NODE_PATH will be used if not defined